cterasdk/edge/directorytree.py

```diff
@@ -0,0 +1,142 @@
+from ..exception import CTERAException
+from ..common import Object
+
+
+class DirectoryTree:
```

Comment:
Since we are returning this class, it needs to be documented as well. So, you need to add docstrings here, add a new file under docs/source/api names cterasdk.edge.directorytree.rst (duplication a similar one and change it) and then add it to the TOC in cterasdk.edge.rst

Reply:
After consideration, I am not the client needs to work with this internal object. The reason I implemented this module in the first place is because we need a common class to handle tree-selections.